src/main/java/mekanism/client/render/hud/MekaSuitHUD.java

```diff
@@ -31,28 +34,46 @@ public void render(ForgeIngameGui gui, PoseStack poseStack, float partialTicks,
         Minecraft minecraft = Minecraft.getInstance();
         if (!minecraft.options.hideGui && !minecraft.player.isSpectator() && MekanismConfig.client.enableHUD.get() && MekanismClient.renderHUD) {
             int count = 0;
-            Map<EquipmentSlot, List<Component>> renderStrings = new LinkedHashMap<>();
+            final List<List<Component>> renderStrings = new LinkedList<>();
```

Comment:
Any reason for using a LinkedList over just using an ArrayList? And at this point it probably should just be a `List<Component>` with using `addAll(list)` below rather than `add(list)`.

Reply:
I thought of that as well, but each component list needs to have a different amount of space between 2 components from the same list (9 vs 2)

Reply:
Ahh missed that part because I didn't expand github's view and missed seeing there is an extra spacer between groups.